joblib/parallel.py

```diff
@@ -54,7 +54,7 @@
 # pool that does not tolerate forking
```

Comment:
This comment will need to be updated to reflect the change. The documentation as well:

https://pythonhosted.org/joblib/parallel.html#bad-interaction-of-multiprocessing-and-third-party-libraries